### PR TITLE
Add 6 icons

### PR DESCRIPTION
**Pull request by lskmn**
# Description
Added 6 new icons.

Not sure about the stroke width for "Finanzguru" & "Tado", but found quite a few icons that use "6px" only – tried a version with 12px but I feel it looks a bit odd & too far from the original.
finanzguru2
finanzguru

### Icons added
* Finanzguru (`de.dwins.financeguru`) - added from icon request form
* Refind (`com.refind.android`)
* Deliveries (`de.orrs.deliveries`) - added from icon request form
* WeNeed (`ch.coop.android.app.shoppinglist`)
* One Jotter (`com.colanotes.android`)
* tado (`com.tado`) - added from icon request form

## Contributor's checklist
- [x] I have followed the [Lawnicons Guidelines](https://github.com/LawnchairLauncher/lawnicons/blob/develop/.github/CONTRIBUTING.md)
- [x] I have ensured that Lawnicons builds correctly
- [x] I am willing to make changes to my icons if someone suggests changes

### CODE REVIEW

File: svgs/refind.svg
Comment (x9136) [inline svgs/refind.svg:1]: There's no problem with your icon. I just want to suggest an alternative, considering the original. You decide what's best.

Screenshot 2023-12-06 at 19 03 05

File: svgs/tado.svg
Comment (x9136) [inline svgs/tado.svg:1]: This icon can be accepted.

<details><summary>FYI</summary></details>
All icons of this type have one problem: they stand out too much from the rest. It takes a lot of time to solve this, so most often long work on one icon is a waste of time, not an improvement of the iconpack.

File: svgs/weneed.svg
Comment (x9136) [inline svgs/weneed.svg:1]: This icon should be redone with a stroke and don't forget the rounded joins.

**Review by x9136 — CHANGES_REQUESTED** [review]

**Comment by lskmn** [comment]
Hey @x9136, thank you very much for the feedback, I hope I'll find the time to implement the changes soon.

File: svgs/weneed.svg
Comment (lskmn) [reply svgs/weneed.svg:1]: Removed it for now, as I don't find the time to remake it.

File: svgs/refind.svg
Comment (lskmn) [reply svgs/refind.svg:1]: Good point, thanks for the feedback. Implemented this version.

File: svgs/one_jotter.svg
[the remark this answers is not in the document]
Comment (lskmn) [reply svgs/one_jotter.svg:1]: Done

File: svgs/finanzguru.svg
[the remark this answers is not in the document]
Comment (lskmn) [reply svgs/finanzguru.svg:1]: Found the big eyes to be a bit creepy tbh, but wen't with some middle ground. :)

File: svgs/deliveries.svg
[the remark this answers is not in the document]
Comment (lskmn) [reply svgs/deliveries.svg:1]: Done, thanks.

**Comment by x9136** [comment]
I've tried fixing the obvious conflicts, but it looks like there are others. Perhaps only updating your branch and recreating the PR will help here.

As for the icons.

Screenshot 2024-01-10 at 06 32 38
Incorrect size (161px in height). Also, I'd like to check how the eyes look on the phone screen, but I can't do that while there are conflicts.

Screenshot 2024-01-10 at 06 34 47
You need to use a stroke instead of a fill.

The others are okay.

**Pull request closed without merging**